znai-server/src/main/java/com/twosigma/znai/server/DocumentationServer.java

```diff
@@ -99,10 +104,18 @@ public HttpServer create() {
                     router.route("/" + params.get("docId") + "/static"), ctx));
         });
 
+        router.delete("/:docId").handler(ctx -> {
+            MultiMap params = ctx.request().params();
+            String docId = params.get("docId");
+            String actor = params.get("actor");
+            new DocumentationRemoveHandler(docId, actor).handle(ctx.request());
```

Comment:
Let’s either use Vertx in the name here or remove it from the one below

Reply:
Removed Vertx.